contracts/test/token/wousd.js

```diff
@@ -3,7 +3,7 @@ const { defaultFixture } = require("../_fixture");
 
 const { ousdUnits, daiUnits, isFork, loadFixture } = require("../helpers");
 
-describe("WOUSD", function () {
+describe.only("WOUSD", function () {
```

Comment:
Remove before merging to master :)

Reply:
<img width="303" alt="image" src="https://user-images.githubusercontent.com/837/161339145-b5124068-fc21-4015-b783-9032a4518f9d.png">

Fixed.